### ISSUE TITLE
ListView Refresh from ListView Command Set

### ISSUE THREAD

**Issue by thechriskent** [SharePoint/sp-dev-docs#818]
#### Category
- [X] Question
- [ ] Typo
- [ ] Bug
- [ ] Additional article idea

Is it possible request the list to refresh using a ListView Command Set? We can always nuke the page with a 
```
location.reload();
```

However, it seems like there should be an API for requesting that the listview retrieve its items again. If this is not currently possible, then I will submit a UserVoice request. Thanks!

**Comment by zhenguang66** [comment]
Yes we're aware of this issue and have put this into our backlog which we'll work on to provide a public API. Thanks for your patience.

**Comment by kachihro** [comment]
This hasn't been updated in a long time.    

Why not include 'officeFabricIconFontName' in the JSON, like WebParts ;
https://blog.mastykarz.nl/specify-icon-sharepoint-framework-client-side-web-part/

**Comment by AJIXuMuK** [comment]
> This hasn't been updated in a long time.
> 
> Why not include 'officeFabricIconFontName' in the JSON, like WebParts ;
> https://blog.mastykarz.nl/specify-icon-sharepoint-framework-client-side-web-part/

unrelated comment
